bodhi/client/__init__.py

```diff
@@ -1009,11 +1011,12 @@ def releases():
 @releases.command(name='create')
 @handle_errors
 @add_options(release_options)
-def create_release(user, password, url, debug, **kwargs):
+def create_release(user, password, url, debug, not_composed_by_bodhi, **kwargs):
```

Comment:
If you like my suggestion above, this becomes:

```suggestion
def create_release(user, password, url, debug, composed_by_bodhi, **kwargs):
```